p2panda-rs/src/test_utils/data.rs

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#![allow(missing_docs)]
```

Comment:
I think it could be nice to have docs here!

Reply:
:+1: